modules/oadp-restic-issues.adoc

```diff
@@ -53,13 +53,13 @@ The `DeploymentConfig` object redeploys the `Restore` pod, causing the `Restore`
 
 .Solution
 
-. Create a `Restore` CR that excludes the `ReplicationController` and `DeploymentConfig` resources:
+. Create a `Restore` CR that excludes the `ReplicationController`, `DeploymentConfig` and `templateinstances.template.openshift.io` resources:
```

Comment:
not critical, but maybe change to `TemplateInstance` instead of `templateinstances.template.openshift.io` here to be aligned with `ReplicationController`, `DeploymentConfig`

Reply:
I meant just TemplateInstances, without `.template.openshift.io`...

Reply:
@mperetzred Is it relevant for all versions of OADP?

Reply:
@mperetzred Yes, this change. Is it relevant for the current version of OADP? Will it be relevant when 1.0.3 is released?

Reply:
@mperetzred 
Done

Reply:
ack